mtools/mplotqueries/mplotqueries.py

```diff
@@ -118,6 +119,11 @@ def __init__(self):
                                     help=("Save the plot to a file instead of "
                                           "displaying it in a window"))
 
+        # SERVER-16176 - checkpoints argument has been added to the existing group
+        self.argparser.add_argument('--checkpoints',
+                                    action='store_true', default=None,
+                                    help=("display the slow transactions "
```

Comment:
It is actually slow WiredTiger checkpoints if I am not mistaken. This is not related to actual transactions.